packages/transaction-controller/src/TransactionController.ts

```diff
@@ -2769,4 +2795,8 @@ export class TransactionController extends BaseControllerV1<
       error?.data?.message?.includes('nonce too low')
     );
   }
+
+  private getGasFeeFlows(): GasFeeFlow[] {
+    return [new LineaGasFeeFlow(), new DefaultGasFeeFlow()];
```

Comment:
I saw that `DefaultGasFeeFlow` is removed since we don't want to use it in the client today. However when it's enabled, the order of this array will be important since we are matching flows with `.find` and it goes from left to right. Is it worth adding some kind of unit test (since it's also private not sure it's possible to read it in tests) to guarantee that the `DefaultGasFeeFlow` is the last member here?

Reply:
Good spot, I've updated the unit tests to assert both the exact list passed to the `GasFeePoller` and the list passed to `updateGasFees`.